### ISSUE TITLE
Failed to load Python shared library

### ISSUE THREAD

**Issue by ihatecorps** [Mord3rca/gamma-launcher#156]
Hello, sorry for the issue, but since 2.0 I get this error when I try to do full-install

[PYI-4975:ERROR] Failed to load Python shared library '/tmp/_MEIEQ6Ax9/libpython3.11.so.1.0': dlopen: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by /tmp/_MEIEQ6Ax9/libpython3.11.so.1.0)

It looked to me like a Python version error so I tried pyenv + 3.11 and 3.12 but I get the same error.

I tried each version with [pyenv shell <version>] command, same error.

Also the directory '/tmp/_MEIEQ6Ax9/libpython3.11.so.1.0' doesn't exist on my drive, but /lib/x86_64-linux-gnu/libm.so.6 does.

1.7 worked and still works (installs everything without issues), but I get a different error when, at the end, I launch the game. Probably because of some different mods?

No idea how to fix! 

I'm a Linux noob, what else can I add that could help you help me? I'm on PoP!_OS 22.4 which has Python 3.10.12 installed

Thank you for the launcher and for any help!

**Comment by Mord3rca** [comment]
Hey !
known issue when using self-extract exec: https://github.com/Mord3rca/gamma-launcher?tab=readme-ov-file#glibc-errors

Small note: When you are in your venv, use `gamma-launcher` and not `./gamma-launcher` to exec the correct script

I cannot help you further if you don't provide what command did you type / output you got.

**Comment by Mord3rca** [comment]
Please use [venv](https://github.com/Mord3rca/gamma-launcher?tab=readme-ov-file#using-pip-from-source) instead, this is what has been tested during development.

You can use other install method etc but this is not an issue for this repo if it is not in the README

**Comment by ihatecorps** [comment]
Appreciate the help, I'll try venv later today! Thanks, have a nice one

**Comment by Mord3rca** [comment]
Also, Pyenv does not provide virtualenv:

> Manage virtualenv. Of course, you can create [virtualenv](https://pypi.python.org/pypi/virtualenv) yourself, or [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) to automate the process.

Source: https://github.com/pyenv/pyenv?tab=readme-ov-file#in-contrast-with-pythonbrew-and-pythonz-pyenv-does-not

**Comment by ihatecorps** [comment]
Done, thanks a lot for your help!

I followed your venv guide and it looks like it's installing (just started downloading Anomaly 1.5.3)!

Thank you again for the launcher and the help, see you in the Zone!

>Also, Pyenv does not provide virtualenv

I didn't know that a virtual environment was needed, as I used to just ./gamma-launcher up until 1.7 so I thought it was only a Python version issue, that's why I went the pyenv route (just did a websearch for "update Python linux", understood that you really can't unless you want to break everything, so searched for alternatives and pyenv popped out and went that way).

Thanks again!
